.github/workflows/conventional-pr-title.yaml

```
@@ -0,0 +1,18 @@
+name: 'PR title follows conventional commits'
+
+on:
+  pull_request_target:
```

I hope there's a Zizmor check at some point in this repo, so perhaps worth adding this in preparation?

```suggestion
  pull_request_target:  # zizmor: ignore[dangerous-triggers] Doesn't touch code - pull_request results in cancellation errors.
```

---

Sure, either's good with me.